src/Server/WPHelper.php

```diff
@@ -23,8 +23,11 @@ class WPHelper extends Helper {
 	 * @throws RequestError
 	 */
 	public function parseRequestParams( $method, array $bodyParams, array $queryParams ) {
+		// Apply wp_unslash to query (GET) variables to undo wp_magic_quotes. We
+		// don't need to do this for POST variables because graphql-php reads the
+		// HTTP body directly.
 		$parsed_body_params = $this->parse_params( $bodyParams );
-		$parsed_query_params = $this->parse_extensions( $queryParams );
+		$parsed_query_params = $this->parse_extensions( wp_unslash( $queryParams ) );
```

Comment:
@chriszarate at one point we addressed this via this PR: https://github.com/wp-graphql/wp-graphql/pull/362/files

Probably didn't get fully translated during the change to the formal Request class? 

Curious what the difference between `wp_unslash` and `wp_kses_stripslashes` are and which one we should use 🤔

Reply:
@jasonbahl Yeah looks like a regression on my part. 😭 

`wp_kses_stripslashes` is not recursive and brings it's own implementation. Seems like `stripslashes` is a better choice since it speaks directly to the source of the issue (`magic_quotes`).

```
function wp_unslash( $value ) {
    return stripslashes_deep( $value );
}
```

```
function wp_kses_stripslashes($string) {
    return preg_replace('%\\\\"%', '"', $string);
}
```

Reply:
nice! LGTM